charts/testkube-api/templates/minio.yaml

```diff
@@ -57,7 +57,7 @@ spec:
             # Name of the PVC created earlier
             claimName: testkube-minio-pv-claim-{{ .Release.Namespace }}
       containers:
-        - name: testkube-minio-{{ .Release.Namespace }}
+        - name: testkube-minio
```

Comment:
will it work for existing minio instance or we will loose previous  data

Reply:
this should only restart the minio pod, data should be on the PVC